### PR TITLE
Set PPIN header

### PR DESCRIPTION
**Pull request by gabrielbriones**
Signed-off-by: Gabriel Briones Sayeg <gabriel.briones.sayeg@intel.com>

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -54,6 +54,7 @@ AC_CHECK_HEADERS([sys/socket.h])
 AC_CHECK_HEADERS([sys/types.h])
 AC_CHECK_HEADERS([syslog.h])
 AC_CHECK_HEADERS([unistd.h])
+AC_CHECK_HEADERS([numa.h])
```
Comment (phmccarty) [inline configure.ac]: Is there a pkgconfig file from libnuma? That might be nicer to check for, because the cflags and linker flags would be autodetected.

**Comment by alexjch** [comment]
Travis build is failing

File: configure.ac (hunk 1 above)
Comment (gabrielbriones) [reply configure.ac]: There is numactl-devel (libnuma-dev for debian)

**Comment by alexjch** [comment]
I was taking a look at the travis failure and I was able to reproduce the error in my system (Clear Linux)

```
PASS: tests/check_probes 13 - ../../tests/check_probes.c:probes:double_fault_payload: Passed
PASS: tests/check_probes 14 - ../../tests/check_probes.c:probes:bad_page_map_payload: Passed
PASS: tests/check_probes 15 - ../../tests/check_probes.c:probes:bug_kernel_handle_payload: Passed
PASS: tests/check_probes 16 - ../../tests/check_probes.c:probes:bug_kernel_handle_payload_new_format: Passed
ERROR: tests/check_libtelemetry - missing test plan
ERROR: tests/check_libtelemetry - exited with status 139 (terminated by signal 11?)
PASS: tests/create-core.sh 1 - found /proc/sys/kernel/core_pattern
SKIP: tests/create-core.sh 2 # SKIP /proc/sys/kernel/core_pattern must contain string "core" to continue
============================================================================
Testsuite summary for telemetrics-client 1.14.2
============================================================================
# TOTAL: 42
# PASS:  39
# SKIP:  1
# XFAIL: 0
```

"signal 11" it's a segmentation fault. My theory is that something is amiss when the system does not have the capability that you are trying to use.

**Comment by alexjch** [comment]
This PR can not be merged due to privacy concerns with information reported.

**Pull request closed without merging**